modules/javafx.graphics/src/test/java/test/javafx/scene/CssStyleHelperTest.java

```diff
@@ -0,0 +1,602 @@
+/*
+ * Copyright (c) 2020, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Minor : As this is a new file being added, there should be just one 2020 (The year in which file has been introduced)